public/styles.css

```diff
@@ -206,3 +206,13 @@ table > tbody > tr > td {
     border-radius: 8px;			
     pointer-events: none;			
 }
+.receivedGrid line {
```

Comment:
a universal class for all the graphs would be cleaner.

Reply:
Yeah but it won't work because I am using the class to remove available gridlines when one clicks on y control button `d3.selectAll(".receivedGrid").remove();` without which the gridlines will fill the graph area.  This explains why each graph gridlines' should be independent of others.